docs/README.md

```diff
@@ -175,26 +196,29 @@ server "variables-srv" {
 }
 ```
 
-### Expressions <a name="expressions">
-Since we use HCL2 for our configuration, we are able to use attribute values as expression:
+### Expressions
+
+Since we use HCL2 for our configuration, we are able to use attribute values as
+expression:
 
 ```hcl
-# Arithmetic with literals and application-provided variables
+// Arithmetic with literals and application-provided variables
```

Comment:
`#` is fine here: https://github.com/hashicorp/hcl2/blob/master/hcl/hclsyntax/spec.md#comments-and-whitespace

Reply:
This was changed to keep the related test simple.